Doc/tools/check-epub.py

```diff
@@ -0,0 +1,23 @@
+import sys
+from pathlib import Path
+
+
+def main() -> int:
+    if not Path("Doc").exists() or not Path("Doc").is_dir():
+        raise RuntimeError("Must run this script from the repo root")
```

Comment:
Let's keep this closer to `Doc/tools/check-warnings.py`. It helps remove text duplication in tracebacks, see https://pypi.org/project/flake8-errmsg/

```suggestion
    wrong_directory_msg = "Must run this script from the repo root"
    if not Path("Doc").exists() or not Path("Doc").is_dir():
        raise RuntimeError(wrong_directory_msg)
```